### ISSUE TITLE
You have tried to use a one-time login link that has either been used

### ISSUE THREAD

**Issue by Corvalan** [omega8cc/boa#1402]
![chrome_OAtAVCh4Tg](https://user-images.githubusercontent.com/8040352/60715101-93dac700-9f1c-11e9-9db4-0cd3a6d11943.png)

This error always appears when I use the one time login link for every new site.
What causes this?

**Comment by omega8cc** [comment]
This is normal if the site either cleared caches via Verify or cron between the moment the link was generated and the moment you have clicked on it.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by Corvalan** [comment]
yes, but that's the point, nothing happened between the link generation and the clicking, it now treats every first attempt as being used/invalid.

**Comment by omega8cc** [comment]
We need more information: the distribution used or at least core versions, plus output of `boa info more` added as Gist.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>
